locale/app.pot

```diff
@@ -0,0 +1,253 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Why are there an app.pot and a app.edit.pot, can't there just be one? This will stay empty forever otherwise